Getting Error java.lang.NoClassDefFoundError: com/google/common/base/Function

**INFO:**

**LINUX VERSION** : 4.15.0-1113-azure
**JDK VERSION** : 
openjdk version "1.8.0_292"
OpenJDK Runtime Environment (build 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10)
OpenJDK 64-Bit Server VM (build 25.292-b10, mixed mode)

ERROR: Getting below error while running cmd - **./bin/connect-standalone config/connect-standalone.properties config/connect-hana-test-source-1.properties config/connect-hana-test-sink-1.properties**

[2022-01-18 17:32:33,680] ERROR WorkerSinkTask{id=hanasink001-0} **Task threw an uncaught and unrecoverable exception. Task is being killed and will not recover until manually restarted (org.apache.kafka.connect.runtime.WorkerTask:184)
java.lang.NoClassDefFoundError: com/google/common/base/Function**
        at com.sap.kafka.connect.sink.hana.HANASinkTask.initWriter(HANASinkTask.scala:29)
        at com.sap.kafka.connect.sink.hana.HANASinkTask.start(HANASinkTask.scala:24)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:308)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:196)
        at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:182)
        at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:231)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: com.google.common.base.Function
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 11 more
[2022-01-18 17:32:33,695] INFO Stopping task (com.sap.kafka.connect.sink.hana.HANASinkTask:62)
[2022-01-18 17:32:33,695] WARN Could not stop task (org.apache.kafka.connect.runtime.WorkerSinkTask:171)
java.lang.NullPointerException
        at com.sap.kafka.connect.sink.GenericSinkTask.stop(GenericSinkTask.scala:63)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.close(WorkerSinkTask.java:169)
        at org.apache.kafka.connect.runtime.WorkerTask.doClose(WorkerTask.java:161)
        at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:187)
        at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:231)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

Can someone help in resolving this?

--- a/examples/persons1/README.md
+++ b/examples/persons1/README.md
@@ -44,7 +44,7 @@ $ make get_libs
 Getting jar files into target ...
 ...
 $ ls target
-ngdbc-2.10.14.jar
+guava-31.0.1-jre.jar  ngdbc-2.10.14.jar
 $
 ```
 

--- a/examples/persons1/libs.txt
+++ b/examples/persons1/libs.txt
@@ -1,1 +1,2 @@
+com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.jar
 com/sap/cloud/db/jdbc/ngdbc/2.10.14/ngdbc-2.10.14.jar

